AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerdetector/DefaultActiveSpeakerDetector.swift

```diff
@@ -42,9 +44,7 @@ typealias DetectorCallback = (_ attendeeIds: [AttendeeInfo]) -> Void
                 strongSelf.policiesAndCallbacks.forEach { (_, policyAndCallback) in
                     strongSelf.speakerScores.forEach { (attendeeInfo, _) in
                         let lastTimestamp = strongSelf.mostRecentUpdateTimestamp[attendeeInfo] ?? 0
```

Comment:
If the values in `mostRecentUpdateTimestamp` are `TimeInterval` now, should we still assign `0` as default value?

Reply:
TimeInterval is just a typealias of Double. This has the same effect as previous value. Previously: Int(0 ms), now: Double(0 s)